dynamic_rest/meta.py

```diff
@@ -36,15 +39,30 @@ def get_model_field(model, field_name):
     """
     meta = model._meta
     try:
-        return meta.get_field_by_name(field_name)[0]
+        if DJANGO110:
+            field = meta.get_field(field_name)
+        else:
+            field = meta.get_field_by_name(field_name)[0]
+        return field
     except:
+        if DJANGO110:
+            related_objs = (
+                f for f in meta.get_fields()
+                if (f.one_to_many or f.one_to_one)
+                   and f.auto_created and not f.concrete
```

Comment:
Unit tests look good, but there seems to be a minor PEP8 style violation on this line:

```
dynamic_rest/meta.py:52:20: E131 continuation line unaligned for hanging indent
```